__fixtures__/base/disallowed.js

```diff
@@ -9,8 +9,20 @@ let things = {
 	f7: (  ) => ['x'],
 	12: '32',
 	ff: xx => xx,
-	gg: ( {fds, }) => 
+	gg: ( {fds, }) =>
         fds,
 };
 
-nonExistent();
```

Comment:
The lack of a newline at the end of the `disallowed` fixture file was intentional (I thought it was a violation, but looks like it is not).

Reply:
It is now :)